src/lib/pubkey/sphincsplus/sphincsplus_common/sphincsplus.h

```diff
@@ -68,6 +66,26 @@ class BOTAN_PUBLIC_API(3, 1) SphincsPlus_PublicKey : public virtual Public_Key {
 BOTAN_DIAGNOSTIC_PUSH
 BOTAN_DIAGNOSTIC_IGNORE_INHERITED_VIA_DOMINANCE
 
+/**
+ * @brief An SLH-DSA private key.
+ *
+ * This class represents an SLH-DSA private key (or a SPHINCS+ Round 3.1 private key).
+ * Supported are all parameter sets defined in FIPS 205, Section 11. Parameter
+ * sets are specified using the Sphincs_Parameter_Set enum and the with a
+ * Sphincs_Hash_Type enum, for example SLH-DSA-SHA2-128s is defined as
+ * Sphincs_Parameter_Set::SLHDSA128Small and Sphincs_Hash_Type::Sha256.
```

Comment:
Huh?

Reply:
>Huh?

I think at least the private key should have at least some documentation at it's class. It had none so far. Or what do you mean?

Reply:
Oh, I see ;)